### PR TITLE
Email Setup Documentation

### PR DESCRIPTION
**Pull request by louisephilardeau**
The instructions for the email documentation setup

### CODE REVIEW

**Comment by santhoshb-msft** [comment]
Please reduce the size of the images.

File: docs/Publisher-Experience.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ The publisher web application is the admin console for the publisher for definin
   * *ValueList*   : Items of the pick list as comma delimited values
   * *Max*         : Maximum value of the integer field
   * *Min*         : Minimum value of the integer field
-  * *DisplaySequence* :Order / sequence in which the field appears along with other fields
+  * *DisplaySequence* : Order / sequence in which the field appears along with other fields
```
Comment (santhoshb-msft) [inline docs/Publisher-Experience.md:41]: is there any change here?

Comment (louisephilardeau) [reply docs/Publisher-Experience.md:41]: Just added a space before "Order" to stay consistent

File: docs/Publisher-Experience.md [hunk 2]
```diff
@@ -68,10 +68,37 @@ The publisher web application is the admin console for the publisher for definin
...
+
+* Set the emails' recipients in **Plans** and mark them as active
+
+![Plan Events](./images/events-activity.png)
+
+* Make sure:
+  * EmailToAddress OR Bcc is not empty (Success Emails on **Plan**)
+  * Cc is added (on the **Email Template**)
+
+* Set the appropriate config value for sending the emails in a particular scenario
```
Comment (santhoshb-msft) [inline docs/Publisher-Experience.md:85]: fix formatting

File: docs/Publisher-Experience.md [hunk 3]
```diff
@@ -68,10 +68,37 @@ The publisher web application is the admin console for the publisher for definin
 
 * Check the box - **Copy to Customer** to include customer email address in the email notifications relevant to the event
 
+## Email Setup
+
+* Make sure the SMTP values are updated in the **Application Config** page
+
+![Application Config Page](./images/config-page.png)
```
Comment (santhoshb-msft) [inline docs/Publisher-Experience.md:75]: can you highlight the smtp settings please

File: docs/Publisher-Experience.md [hunk 4]
```diff
@@ -68,10 +68,37 @@ The publisher web application is the admin console for the publisher for definin
...
+
+* Set the emails' recipients in **Plans** and mark them as active
+
+![Plan Events](./images/events-activity.png)
+
+* Make sure:
```
Comment (santhoshb-msft) [inline docs/Publisher-Experience.md:81]: Can you specify where these exist

File: docs/Publisher-Experience.md [hunk 5]
```diff
@@ -38,7 +38,7 @@ The publisher web application is the admin console for the publisher for definin
   * *ValueList*   : Items of the pick list as comma delimited values
   * *Max*         : Maximum value of the integer field
```
Comment (santhoshb-msft) [inline docs/Publisher-Experience.md:39]: Could you please add a section bullet item on the top of the page please.

**Review by santhoshb-msft — APPROVED** [review]
Looks good, approving the PR

**Pull request merged**